docs/commands/bootstrap.mdx

```diff
@@ -101,6 +101,12 @@ your `melos.yaml` exists in a package, the dependency version in this
 package will be updated to match the version defined in your
 bootstrap config every time `melos bootstrap` is run.
 
+When there's a need to override the dependency_overrides
+specified in a package's pubspec.yaml, use dependency_overrides in Melos' configuration.
+This allows you to specify different package versions than those defined in the package's
+own dependency overrides. This is especially useful for situations where you're actively
+developing a package that relies on a path dependency not present in the main project.
```

Comment:
I think the use-case could be removed, since there are lots of use-cases for this (I would guess git overrides are more common than path overrides within mono-repos for example?).

```suggestion
own dependency overrides.
```